azure-pipelines-product-construction-service.yml

```diff
@@ -0,0 +1,58 @@
+# Changes the format of the Build.BuildNumber variable
+name: $(Date:yyyyMMdd)$(Rev:r)
```

Comment:
Added the non production trigger. The PR triggers are configured in the repo settings, and it's currently off

Reply:
It is still better to add `pr: none`. For instance, this will work for internal PRs in AzDO if we ever make some.

It's also explicit when reading the YAML - that we don't expect this to run on PRs. So you don't need to go in AzDO and search for the pipelines etc